docs/en/pid-eaa-issuance.rst

```diff
@@ -1101,7 +1173,7 @@ Below is a non-normative example of an Entity Configuration containing an `openi
           },
 
           "openid_relying_party": {
-            <This is the metadata of the PID Provider acting as a Relying Party in the national digital identity framework (CIE/SPID). See spid-cie-oidc-docs for details.>
+            <This is the metadata of the EAA Provider acting as a Relying Party in the national digital identity systems (CIE/SPID). See spid-cie-oidc-docs for details.>
```

Comment:
@peppelinux , why did you replace PID with EAA here? This is the example of EC of a PID Provider.

Reply:
ok